{"uuid":"4fc1e6ca-509a-436c-8341-cfeb2c34c9c1","name":"Scenario Outline: The user getting error message with invalid python code for Introduction page","children":["0106e7bd-4baf-4b5c-9199-602cb47be179"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"5c24ddce-d49e-45db-9934-4b71ab6005f2-attachment","type":"image/png"},{"name":"Myscreenshot","source":"26143101-a11d-4f7d-9d2b-99cbe6efc891-attachment"}],"parameters":[],"start":1696885802336,"stop":1696885802382},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885802383,"stop":1696885802383},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885802383,"stop":1696885802384},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885802384,"stop":1696885802384}],"start":1696885802333,"stop":1696885802385}